core/tests/protractor_desktop/profileFeatures.js

```diff
@@ -60,7 +60,7 @@ describe('Un-customized profile page', function() {
   it('displays no photo, default bio, and no interests when logged out',
```

Comment:
Thanks for finding this @Showtim3! Now that the test is being corrected, I think we should also update the test name to indicate that the photo should be displayed

Reply:
Ok Done!